opal/mca/pmix/external/pmix_ext.c

```diff
@@ -3,7 +3,7 @@
  * Copyright (c) 2014-2015 Intel, Inc.  All rights reserved.
  * Copyright (c) 2014-2015 Research Organization for Information Science
  *                         and Technology (RIST). All rights reserved.
- * Copyright (c) 2014-2015 Mellanox Technologies, Inc.
+ * Copyright (c) 2014      Mellanox Technologies, Inc.
```

Comment:
In order to sync the external glue code to the internal one, I did a simple copy of the internal to the external, and then updated the names to the new component. So the copyrights and everything else just flowed across.